src/utils/apiRootPage.js

```
@@ -0,0 +1,119 @@
+    ok: true,
+    docs: DOCS_URL,
+    studio: local ? 'http://localhost:2021' : HOSTED_STUDIO_URL,
+    ...(local ? { storefront: 'http://localhost:4087' } : {}),
+  };
+};
+
+const escapeHtml = (value) =>
+  String(value)
+    .replace(/&/g, '&amp;')
+    .replace(/</g, '&lt;')
+    .replace(/"/g, '&quot;');
```

ESLint

[error] Replace `⏎··String(value)⏎····.replace(/&/g,·'&amp;')⏎····.replace(/</g,·'&lt;')⏎····` with `·String(value).replace(/&/g,·'&amp;').replace(/</g,·'&lt;')`

(prettier/prettier)